Assets/Source/Entities/Behaviors/Commander.cs

```diff
@@ -68,11 +79,17 @@ void Update()
 				else if (Physics.Raycast(Camera.main.ScreenPointToRay(Input.mousePosition), out hitInfo, Mathf.Infinity, Layers.Map))
 				{
 					// Spawn a marker
-					if (!tempCircle)
-						tempCircle = GameObject.CreatePrimitive(PrimitiveType.Sphere);
+					if (!cursor)
```

Comment:
I kinda prefer using braces on all parts of if-elseif-else chain if they are necessary in any part of the chain. E.g. change to:
```csharp
if(!cursor)
{
	cursor = (GameObject)Instantiate(Resources.Load("Cursor"));
}
else
{
	Destroy (cursor);
	cursor = (GameObject) Instantiate(Resources.Load("Cursor"));
}
```

That being said I think it is fine to omit them if the whole thing is one-liners:

```csharp
if(condition)
	Statement();
else
	OtherStatement();
```